src/app/clusters/ota-requestor/GenericOTARequestorDriver.cpp

```diff
@@ -337,14 +343,14 @@ void GenericOTARequestorDriver::SendQueryImage()
 
     // Default provider timer only runs when there is no ongoing query/update; must stop it now.
     // TriggerImmediateQueryInternal() will cause the state to change from kIdle
-    StopDefaultProviderTimer();
+    StartSelectedTimer(SelectedTimer::kWatchdogTimer);
```

Comment:
Please remove this as the watchdog timer will be set again in `HandleIdleStateExit` when we transition out of of idle state.

Reply:
Done.